Advanced-Recruit-Training/src/BatteryStateDisplayService.cpp

```diff
@@ -24,20 +24,19 @@ BatteryStateDisplayService::~BatteryStateDisplayService()
 void BatteryStateDisplayService::handleBatteryDataReceived(const BatteryData& batteryData)
 {
     batteryStateOfChargeService_.addData(batteryData);
-
-    QTextStream(stdout) << "Voltage: " << batteryData.voltage
-        << " Current: " << batteryData.current
-        << " Total Ah used: " << batteryStateOfChargeService_.totalAmpHoursUsed() << endl;
-
-    // TODO: Print out time until the battery is fully charged or depleted.
-
     QString Time = batteryStateOfChargeService_.timeWhenChargedOrDepleted().toString();
```

Comment:
Pass the format of how the string should be through as the argument, as per http://doc.qt.io/qt-5/qtime.html#toString